src/main/java/de/retest/recheck/suite/flow/ApplyChangesToStatesFlow.java

```diff
@@ -22,6 +22,7 @@
 	private static final Logger logger = LoggerFactory.getLogger( ApplyChangesToStatesFlow.class );
 
 	private final GoldenMasterProvider goldenMasterProvider;
+	private final List<String> filenames = new ArrayList<>();
```

Comment:
`filenames` of what? 😉 Without context, this is not self-explanatory. I suggest something like `missingGoldenMastersFilenames`.